titus-supplementary-component/task-relocation/src/main/java/com/netflix/titus/supplementary/relocation/connector/NodePredicates.java

```diff
@@ -64,11 +65,39 @@ static boolean hasBadCondition(V1Node node, Function<String, Matcher> badConditi
         return false;
     }
 
+    @VisibleForTesting
+    static boolean hasBadTaint(V1Node node, Function<String, Matcher> badTaintExpression,
+                                   int nodeTaintTransitionTimeThresholdSeconds) {
+        if (node.getSpec() != null && node.getSpec().getTaints() != null) {
+            return node.getSpec().getTaints().stream()
+                    .anyMatch(v1Taint -> badTaintExpression.apply(v1Taint.getKey()).matches() &&
+                            matchesTaintValueIfAvailable(v1Taint, Boolean.toString(true)) &&
```

Comment:
I prefer `Boolean.TRUE.toString()` over `Boolean.toString(true)`. I also use a static import to get rid of the `Boolean`.